src/store/app/app.effects.ts

```diff
@@ -866,6 +867,8 @@ export const checkNotificationsPermissions = async (): Promise<boolean> => {
     normalized === RESULTS.GRANTED || normalized === RESULTS.LIMITED;
   if (granted) {
     Braze.requestPushPermission();
+    if (Platform.OS === 'ios')
```

Comment:
Missing token happens on iOS and Android. We shouldn't limit the same functionality to only iOS

Reply:
ok, I haven't checked the Android case yet. We could do that in a separate pull request.